src/client/pages/OfferNew/utils.ts

```diff
@@ -544,3 +571,25 @@ export const getInsuranceTypesFromBundleVariant = (
 export const getQuoteIdsFromBundleVariant = (
   bundleVariant: QuoteBundleVariant,
 ) => bundleVariant.bundle.quotes.map((quote) => quote.id)
+
+export const getTypeOfContractFromBundleVariant = (
+  bundleVariant: QuoteBundleVariant,
+) => bundleVariant.bundle.quotes.map((quote) => quote.typeOfContract)
+
+function allBundleVariantsHaveSameInsuranceType(
+  bundleVariants: Array<QuoteBundleVariant>,
+) {
+  if (bundleVariants.length <= 1) {
+    return false
+  }
+
+  const insuranceType = bundleVariants?.[0].bundle.quotes?.[0]?.data.type
```

Comment:
guess we don't have a selector for this? 😅

Reply:
Nope, but I can add one!